canonicals.js

```diff
@@ -0,0 +1,53 @@
+var APP = APP || {};
```

Comment:
I'd love to dig into some ES6 but I think having a build step makes it way more cumbersome to add new question ids to the canonical list. Being able to make a PR with a new question directly through github's web interface increases the chance it will be done dramatically. With a build step PRs also has to include re-transpiled source code.